// _@vuepress_core@1.3.1@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Volumes/workspace/codespace/github/moli/node_modules/_@vuepress_core@1.3.1@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-c472676e",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-c472676e").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-343bdc6d",
    path: "/component/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-343bdc6d").then(next)
    },
  },
  {
    path: "/component/index.html",
    redirect: "/component/"
  },
  {
    name: "v-45843b12",
    path: "/component/guide/guide.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-45843b12").then(next)
    },
  },
  {
    name: "v-768bab12",
    path: "/component/basic/input.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-768bab12").then(next)
    },
  },
  {
    name: "v-19e438d2",
    path: "/component/about.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-19e438d2").then(next)
    },
  },
  {
    name: "v-6c832670",
    path: "/component/basic/form.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6c832670").then(next)
    },
  },
  {
    name: "v-1643abd5",
    path: "/theme/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1643abd5").then(next)
    },
  },
  {
    path: "/theme/index.html",
    redirect: "/theme/"
  },
  {
    name: "v-1fab5258",
    path: "/component/basic/button.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1fab5258").then(next)
    },
  },
  {
    name: "v-128cda88",
    path: "/component/basic/notice.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-128cda88").then(next)
    },
  },
  {
    name: "v-4d9332e0",
    path: "/component/guide/introduction.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4d9332e0").then(next)
    },
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
